Bronco_Tree/tree.pde

```diff
@@ -51,16 +54,22 @@ class Tree {
     // Function: grow
     // Description: This function is the heart of the algorithm. grow() looks at each
     // 		    leaf and
-    for (Leaf l : leaves) {
```

Comment:
So "leaf" is defined as "a point to which a branch can grow"? It's a bit confusing with the fact that leaves can also be little parts of the tree (we have blossoms) but I think it's fine as long as we're consistent in how we use it.